pkg/clusters/addons/kong/builder.go

```diff
@@ -36,6 +41,34 @@ func (b *Builder) WithPostgreSQL() *Builder {
 	return b
 }
 
+// WithEnterprise deploying kong enterpise
+func (b *Builder) WithEnterprise() *Builder {
+	b.enterprise = true
+	b.repo = DefaultEnterpriseImageRepo
+	b.tag = DefaultEnterpriseImageTag
+	b.license = KongEnterpriseTestLicense
+	b.kongAdminPassword = EnterpriseKongAdminDefaultPWD
+	return b
+}
+
+// WithImage specify docker image repo and tag
+func (b *Builder) WithImage(repo, tag string) *Builder {
+	b.repo = repo
+	b.tag = tag
+	return b
+}
```

Comment:
Similar to previous comments, this nomenclature used for this method is ambiguous: as it's currently written this method provides the values for the Kong proxy container, not the Controller or any other components. We should make the method name more specific to help reduce ambiguity, and we should improve the `godoc` comment for the same reason:

```suggestion
// WithProxyImage configures the container image and tag for the Kong proxy container.
func (b *Builder) WithProxyImage(image, tag string) *Builder {
	b.repo = image
	b.tag = tag
	return b
}
```

It's sort of a nitpick, but I also suggested changing the name to `image` because that's consistent with the method name.

Reply:
will keep the image as it is